src/app/blog/agentic-search/index.mdx

```diff
@@ -0,0 +1,143 @@
+import blogMetadata from "./metadata.json";
+import { AuthorSection } from "@/components/AuthorSection";
+import { HeroImage } from "@/components/HeroImage";
+import { Title } from "@/components/Title";
+import heroImage from "./images/hero.svg";
+import { Note } from "@mintlify/components"
+
+<Title metadata={blogMetadata} />
+<AuthorSection metadata={blogMetadata} />
+<HeroImage src={heroImage} metadata={blogMetadata} />
+
+
+
+Search systems were built for real people. For decades, they were tuned for short keyword queries, low latency, and a concise ranked list of results that a human could read and act on. 
+
+Agents are a newer consumer of search, and they do not behave this way. They iteratively issue structured queries, gather result sets, and read entire documents as part of longer reasoning loops. They can already make decisions; what they lack are interfaces that expose enough structure to make *good* ones.
+
+Most search APIs still treat agents like impatient humans: query in, ranked list out. There is no visibility into why a particular document ranked higher or how to modify that outcome. When an agent produces poor results, it has no insight into whether the query was underspecified, the weights were imbalanced, or the wrong signal dominated. Everything happens inside a black box.
+
+That opacity was once a design goal. Search for humans is meant to feel a little magical, an interface that hides its mechanics behind the screen delivering relevance as a service. But agents have no use for magic. They need transparency, determinism, and control. 
+
+<Note> 
+We aren’t selling agentic search, and we don’t have an agentic search product. We do firmly believe that however agentic search falls out the cornerstone will still be BM25, and that’s one of the reasons why we have built a better full-text search experience (including BM25) in PostgreSQL.
+</Note>
+
+## SQL: The Missing Interface Layer
+
+Agents don’t need a new retrieval algorithm, or even a catalog of tools which they can use; they need a way for retrieval to participate in their reasoning process. To analyze the data available, run queries, see scores, adjust weights, and combine evidence as they collect data-points to influence their next action.
+
+Crucially, they already speak the language for it. Large language models have absorbed SQL from the same public sources that taught humans for years: Stack Overflow, database documentation, open-source repositories, and Reddit discussions explaining joins and indexes. They have not only learned to *write* SQL but to *reason* about how it works; to understand schemas, constraints, and data relationships.
+
+That makes SQL a natural interface for search. It is not a new abstraction that agents must learn; it’s one they already understand, grounded in explicit semantics and predictable structure. Exposing search through SQL doesn’t feel new, it feels like meeting agents where they already are.
```

Comment:
should this paragraph be rephrased and come before the one prior to it. here we are laying a ground truth and above justifying why SQL is ideal.. @jamessewell wdyt?